V0.5.2BeBuilder/bebuilder.rdef

```diff
@@ -0,0 +1,70 @@
+
+resource app_signature "application/x-vnd.bebuilder";
+
+resource app_version {
+	major  = 0,
+	middle = 0,
+	minor  = 0,
```

Comment:
That's funny!
I started the review when the PR was open, and when I finished it the PR had been merged 7 minutes earlier..  but this did not prevent my review from being saved ;-)